src/features/profile/helpers/useChangeEmailMutation.ts

```diff
@@ -46,8 +46,8 @@ export const useChangeEmailMutation = ({
         navigateToProfile()
         analytics.logSaveNewMail()
       },
-      onError: (error: ApiError | unknown) => {
-        onEmailChangeError((error as ApiError)?.content?.code)
+      onError: (error: unknown) => {
+        onEmailChangeError(isApiError(error) ? error.content.code : undefined)
```

Comment:
`code` est toujours défini quand c'est une API ?

Reply:
c'est un `any` https://github.com/pass-culture/pass-culture-app-native/blob/b9098a0eed2b588d449a837f529337af6f472908/src/api/apiHelpers.ts#L250

Reply:
Donc on peut faire ça si jamais `content` est `undefined` ?
```suggestion
        onEmailChangeError(isApiError(error) ? error.content?.code : undefined)
```